features/contain-size.yml

```
@@ -0,0 +1,6 @@
+name: Size containment
+description: "The `contain: size` CSS declaration sets an element's dimensions exclusively by its `height` and `width` properties, ignoring its contents and descendants. This permits the browser to avoid slower layout calculations."
+spec: https://drafts.csswg.org/css-contain-2/#containment-size
+group: containment
+compat_features:
+  - css.properties.contain.size
```

Since I did https://github.com/web-platform-dx/web-features/pull/1290 to us, should we move `css.properties.contain` into here?

```suggestion
  - css.properties.contain
  - css.properties.contain.size
```

---

Yes, let's put it here.